.github/workflows/test-and-publish.yml

```diff
@@ -0,0 +1,150 @@
+name: Run tests and publish Python distribution to PyPI
+
+on:
+  [push, pull_request]
+
+env:
+  DATABASE_SCHEMA: 4.1.0
```

Comment:
You probably want the latest `DATABASE_SCHEMA` version, 4.2.1.

Reply:
Good point. Just done that